less/mixins.less

```diff
@@ -1 +1,32 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+.hidden(){
+    display: none;
+  }
+
+  
+  // Parametric Mixin
+.custom-btn(@bgcolor, @font-color, @br, @fs) {
+    color: @font-color;
+    background-color: @bgcolor;
+    border-radius: @br;
+    font-size: @fs;
+    width: 200px;
+    height: 50px;
+  
+    &:hover {
+      background-color: @font-color;
+      color: @bgcolor;
+    }
+  }
+
+
+  #signup{
```

Comment:
Why is your signup button styled in your mixins file? This should be in the home-page file where you styled the rest of the elements on your home-page.